modules/initramfs/initramfs.sh

```diff
@@ -9,12 +9,11 @@ fi
 
 if [[ "${OS_VERSION}" -le 40 ]]; then
   echo "This module is only compatible with Fedora 41+ images."
-  echo "While it is possible to make this module work with some older Fedora versions, by installing now-depreciated cliwrap,"
-  echo "It is an unsupported functionality, which we don't want to enable."
-  echo "If you wish to make this work, you would need to install cliwrap manually:"
-  echo "rpm-ostree cliwrap install-to-root /"
-  echo "And modify dracut's command to cliwrapped one in module's script:"
-  echo "/usr/libexec/rpm-ostree/wrapped/dracut"
+  exit 1
+fi
+
+if [[ -f "/usr/libexec/rpm-ostree/wrapped/dracut" ]]; then
+  echo "This module is not compatible with images which install depreciated rpm-ostree cliwrap"
```

Comment:
should be "...images that install deprecated rpm-ostree cliwrap"

Reply:
removed the message, as I use cliwrapped dracut if available now.